packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -839,6 +839,42 @@ For right-to-left (RTL) languages, this is flipped.
 </ComboBox>
 ```
 
+### Help text
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Help-text-(description-and-error-message))
+
+Both a description and an error message can be supplied to a ComboBox. Description shows all the time except when the `validationState` is "invalid". The error message should help the user fix their input quickly and it should be changed depending on the detected error. Both the description and error message should be localized as well.
```

Comment:
What do you think of this ^?

Reply:
@dannify which do you like more? you gave two suggestions.
I think I like the first one more.

Reply:
oh weird, not sure what happened there. Should have been the first comment only.